### PR TITLE
Add use cases

### PR DESCRIPTION
**Pull request by aslam341**
Also fixes the following format error for spacing:
image

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -439,87 +439,748 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
-    * 3a1. TaskHub shows an error message.
+* 2c. User enters an invalid value for a field.
+  * 2c1. TaskHub shows an error message.
 
-      Use case resumes at step 2.
+    Use case ends.
 
-**Use case 3: Add an employee**
+* 3d. User edits the project's name to one that already exists.
```
Comment (antonTan96) [inline docs/DeveloperGuide.md:666]: 2d

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -439,87 +439,748 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+* 2d. User enters multiple values for a field.
+  * 2d1. TaskHub shows an error message.
+
+    Use case ends.
+
+* 2e. User enters an employee that is already assigned to the task in the project.
+  * 2e1. TaskHub shows success message but there is no effect on the project or task.
+
+    Use case ends.
```
Comment (antonTan96) [inline docs/DeveloperGuide.md:1112]: Maybe this could be Use case resumes from step 4?

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (aslam341) [reply docs/DeveloperGuide.md]: You are correct, it is supposed to be 2d.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (aslam341) [reply docs/DeveloperGuide.md]: Sure, I have changed it.

**Review by Chandan8186 — APPROVED** [review]
LGTM, there are a couple of minor bugs but I'll fix them in a separate PR

**Pull request merged**